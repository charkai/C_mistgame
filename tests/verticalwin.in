place A19
place D8
place A14
place D9
place B4
place D10
place B9
place D7
place C2
place D11
