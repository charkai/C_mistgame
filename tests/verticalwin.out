White wins!
A19D8A14D9B4D10B9D7C2D11
Thank you for playing!
